packages/SwingSet/src/kernel/state/kernelKeeper.js

```diff
@@ -593,9 +589,7 @@ export default function makeKernelKeeper(storage, kernelSlog) {
   function getVatIDForName(name) {
     assert.typeof(name, 'string');
     const k = `vat.name.${name}`;
-    if (!storage.has(k)) {
-      throw new Error(`vat name ${name} must exist, but doesn't`);
-    }
+    assert(storage.has(k), X`vat name ${q(name)} must exist, but doesn't`);
```

Comment:
I don't think we need the `q()` here, we aren't using it anywhere else, and we've already asserted that `name` is a string.

Reply:
That was due only to
https://github.com/Agoric/agoric-sdk/blob/667397754fc3d4db5956a340ef78b94e030a9701/packages/SwingSet/test/test-devices.js#L539

Since I didn't know for sure what such tests were testing, I preserved the information. I'll fix it.

Reply:
ah, yes, let's leave the `q()` and aim to do a later cleanup pass if/when `t.throws` can handle it

Reply:
Got rid of that particular `q`. Taking your advice on the others --- leave it till later.